### PR TITLE
Iss815

### PR DESCRIPTION
**Pull request by KirbyKatcher**

### CODE REVIEW

File: galasa-managers-parent/galasa-managers-core-parent/dev.galasa.core.manager/src/main/java/dev/galasa/core/manager/internal/CoreManagerImpl.java [hunk 1]
```diff
@@ -222,6 +223,14 @@ public ICredentials getCredentials(@NotNull String credentialsId) throws CoreMan
...
 		}
 	}
+	
+	@Override
+	public ICredentialsUsernamePassword getUsernamePassword(@NotNull String credentialsId) throws CoreManagerException {
+		ICredentials cred =getCredentials(credentialsId);
+		if(!(cred instanceof ICredentialsUsernamePassword)) {
+			throw new CoreManagerException("Unable to retrieve username/password credentials for id"+ credentialsId);
+		}
+	}
```
Comment (hobbit1983) [inline galasa-managers-parent/galasa-managers-core-parent/dev.galasa.core.manager/src/main/java/dev/galasa/core/manager/internal/CoreManagerImpl.java:233]: Don't we need to return the cred object back to the caller?

**Review by hobbit1983 — CHANGES_REQUESTED** [review]

**Comment by hobbit1983** [comment]
Manually set DCO to pass - KirbyKatcher is new to Galasa team and this code is fine to be delivered under the EPLv2 License

**Review by hobbit1983 — APPROVED** [review]

**Comment by hobbit1983** [comment]
@KirbyKatcher any reason why we haven't merged this yet?

**Comment by galasa-team** [comment]
Build successful

**Comment by KirbyKatcher** [comment]
No reason, merged now

**Pull request merged**